presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClientModule.java

```diff
@@ -16,19 +16,33 @@
 import com.facebook.presto.plugin.jdbc.BaseJdbcConfig;
 import com.facebook.presto.plugin.jdbc.JdbcClient;
 import com.google.inject.Binder;
-import com.google.inject.Module;
 import com.google.inject.Scopes;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
 
+import java.net.URI;
+
+import static com.google.common.base.Preconditions.checkState;
 import static io.airlift.configuration.ConfigBinder.configBinder;
 
 public class MySqlClientModule
-        implements Module
+        extends AbstractConfigurationAwareModule
 {
     @Override
-    public void configure(Binder binder)
+    protected void setup(Binder binder)
     {
         binder.bind(JdbcClient.class).to(MySqlClient.class).in(Scopes.SINGLETON);
-        configBinder(binder).bindConfig(BaseJdbcConfig.class);
+        BaseJdbcConfig config = buildConfigObject(BaseJdbcConfig.class);
```

Comment:
Good point. I was thinking we don't have `mysqldriver` in `compile` scope.
That's much better now, thanks!